src/pages/member/QuickRegister.js

```diff
@@ -103,6 +108,13 @@ const EventFormContainer = (props) => {
     diet: Yup.string().required('Dietary restriction is required')
   })
 
+  const validationNonUBCStudent = Yup.object({
```

Comment:
technically this is still a schema
```suggestion
  const validationSchemaNonUBCStudent = Yup.object({
```